CMakeLists.txt

```diff
@@ -83,7 +83,7 @@ option(
 
 set(AVIF_USE_CXX OFF)
 
-if(AVIF_LOCAL_LIBGAV1)
+if(AVIF_BUILD_APPS OR AVIF_ENABLE_FUZZTEST OR AVIF_ENABLE_GTEST OR AVIF_LOCAL_LIBGAV1)
```

Comment:
I know this was already the case but do you know why we enable c++ when AVIF_BUILD_APPS is on? current apps are C only afaik

Reply:
Nope, I was wondering too. I kept it in case there is some experimental stuff I did not know about ...
Let me remove it.

Reply:
Mh that's kind of weird, either are_images_equal  should not be built when only AVIF_BIULD_APPS is on, or it should be moved to the apps/ directory I think...  But that's a problem for another PR